packages/babel-preset-es2015/src/index.js

```diff
@@ -1,42 +1,50 @@
 module.exports = function(context, opts) {
-  /* eslint-disable no-var */
-  var loose = false;
-  var modules = true;
-  /* eslint-enable no-var */
+  const moduleTypes = ["commonjs", "amd", "umd", "systemjs"];
+  let loose = false;
+  let modules = "commonjs";
+  let spec = false;
```

Comment:
Was there a specific motivation for adding this? I ask because I'm not entirely convinced we get much from it.